dapp/src/utils/orangekit/ledger-live/bitcoin-provider.ts

```diff
@@ -274,7 +274,7 @@ export default class AcreLedgerLiveBitcoinProvider
       message: {
         ...data,
         operation: data.operation.toString(),
-        nonce: data.nonce.toString(),
+        nonce: data.nonce.toString(16),
```

Comment:
Does it add `0x` prefix?

Reply:
Regardless of what the underlying library does, we should output the value in an unambiguous format. This is especially true here since one of the numbers (`operation`) is handled as a decimal string and another (`nonce`) as a hexadecimal.

Maybe we could use `ethers.utils.hexlify`?

Reply:
Should we pass just converted number to hex with `0x` prefix, or add leading zeros? For example `12`:
1. `0xc`
2. `0x0c`
3.  `0x000000000000000000000000000000000000000000000000000000000000000c`

Reply:
925fc87